src/acceptance/config/config.go

```diff
@@ -55,6 +55,7 @@ type Config struct {
 
 	ASApiEndpoint          string `json:"autoscaler_api"`
 	ServiceOfferingEnabled bool   `json:"service_offering_enabled"`
+	EnableServiceAccess    bool   `json:"enable_service_access"`
```

Comment:
Can you please update the [AutoScaler UAT Guide](https://github.com/cloudfoundry/app-autoscaler-release/blob/master/src/acceptance/README.md#test-configuration) and describe how to use the new flag `enable_service_access`?

Reply:
Resolved in 91d94d3